include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1381,6 +1381,36 @@ struct OrtApi {
                   _In_ const void* model_data, size_t model_data_length,
                   _In_ const OrtSessionOptions* options, _Inout_ OrtPrepackedWeightsContainer* prepacked_weights_container,
                   _Outptr_ OrtSession** out);
+
+  /**
+  * Use this API to create the configuration of a TensorRT Execution Provider.
+  */
+  ORT_API2_STATUS(CreateTensorRTProviderOptions, _Outptr_ OrtTensorRTProviderOptions** out);
```

Comment:
Please, document all the params and state how to destroy the instance when done.

Reply:
added. and move the definitions to tensorrt_provider_factory.h